### DISCUSSION TITLE
How to update a service to latest version

### DISCUSSION THREAD

**Discussion by brasilikum** [coollabsio/coolify#760] [Q&A]
Hi,
I am facing a bug in n8n that fixed in the latest version of n8n.

As I looked around in n8n, I saw that there are only some versions available. Looking at the code,
there seems to be a fixed list of versions per service.

I would like to understand why coolify decided for this design (vs for example allowing any tag that is valid).
I would also like to know if there is any way I can manually update n8n.

Best
G

**Comment by paulocoutinhox** [comment]
This is my problem too:
https://github.com/coollabsio/coolify/issues/987
